api/inc/vmpu_exports.h

```diff
@@ -143,13 +143,29 @@ typedef struct
     UvisorBoxAcl acl;
 } UVISOR_PACKED UvisorBoxAclItem;
 
+/*
+ * Privileged system interrupt hooks
+ *
+ * In this version of uVisor, uVisor lives alongside an RTOS that requires
+ * running privileged code. In order for the RTOS to run any privileged
+ * code, uVisor must allow the RTOS to handle privileged system interrupts.
+ * Only the following system interrupts are hookable. Code called by these
+ * handlers circumvents uVisor security. HANDLE WITH CARE. */
+typedef struct
+{
```

Comment:
Our established style in this file appears to differ from your proposal. Shall I change them all?